docgen/src/examples/getting-started-boilerplate/index.html

```diff
@@ -41,7 +41,7 @@
     </div>
 
     <!-- js -->
-    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.0.0/dist/instantsearch.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.0.0"></script>
```

Comment:
Can you also update the version of IS.js used by the boilerplate?

Reply:
You mean the CodeSandbox one? We can only do so when the lib is actually re-released and picked up by jsDelivr

Reply:
No I mean this exact line 😅 Which is the source for the boilerplate but has a fixed version number.